src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobResumeBuild.java

```diff
@@ -0,0 +1,45 @@
+package com.tikal.jenkins.plugins.multijob;
+
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Cause;
+import hudson.model.ProminentProjectAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import java.io.IOException;
+
+public class MultiJobResumeBuild implements ProminentProjectAction {
```

Comment:
NiT: as I see the plugin mostly uses spaces for the tabulation. It would be better to follow this approach